deploy/helm/templates/tooljet/service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: tooljet
+spec:
+  type: ClusterIP
```

Comment:
@replicadse Does it make sense to change this to a load balancer such that it can be publicly available?

Reply:
@akshaysasidrn I see what you mean.
I do not think (and in fact really hate it) when the default is a LoadBalancer. Why? In any cloud environment, this will generate a new static IP for some kind of cloud managed LB. This adds extra cost plus works around other things you have already set up maybe. Many people use traefik as reverse proxy and it handles TLS whereas a service directly pinned to an LB does not.
But: One thing we usually encounter is that the service type is configurable from the `values.yaml` file. Maybe we don't need to make that decision but let the user decide which service type he/she wants? Update follows...

Reply:
Update in commit [ac90f18](https://github.com/ToolJet/ToolJet/pull/2393/commits/ac90f182c5e41791315a26ced854dbd9bdc748f2)